server/bus/compare_manifest.py

```diff
@@ -0,0 +1,42 @@
+import json
+import sys
+
+station_stops = {}
+
+
+def runone(path, first=False):
+    unchanged = True
+    current = json.load(open(path))
+    for i in list(current.values())[0]['stations']:
+        s = i['station']
+
+        if s not in station_stops:
+            station_stops[s] = {}
+            if not first:
+                print("Found missing station %s" % s)
```

Comment:
```suggestion
                print("  + Found new station %s" % s)
```
This and the below indentation might help readability without needing to get pprint involved (though that's also an option)

I'd love it if this script could also detect which existing stations/stops were removed. It would really help clarify some situations like the nubn -> dudly deal (dudly and 1-1-64 aren't actually new, it's just renamed).

Reply:
Done.